packages/core/contracts/governance/MetaHumanGovernor.sol

```diff
@@ -0,0 +1,572 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import '@openzeppelin/contracts/governance/Governor.sol';
+import '@openzeppelin/contracts/governance/extensions/GovernorSettings.sol';
+import '@openzeppelin/contracts/governance/extensions/GovernorVotes.sol';
+import '@openzeppelin/contracts/governance/extensions/GovernorVotesQuorumFraction.sol';
+import '@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol';
+import './CrossChainGovernorCountingSimple.sol';
+import './DAOSpokeContract.sol';
+import './wormhole/IWormholeRelayer.sol';
+import './wormhole/IWormholeReceiver.sol';
+import './magistrate/Magistrate.sol';
+import 'hardhat/console.sol';
```

Comment:
Remove console since it's used for debugging purpose.

Reply:
removed